analyzer/src/testFixtures/kotlin/Extensions.kt

```diff
@@ -114,3 +119,26 @@ fun Spec.analyze(
 
     return analyzer.analyze(managedFiles).withResolvedScopes()
 }
+
+fun Spec.create(
```

Comment:
Have you considered `reified` or some other way to be resilient about manager name changes? One could use the Factory class as type parameter, but the name is longer...probably that's not so nice.

Reply:
I could not think of a good way unfortunately.